banners.yml

```diff
@@ -0,0 +1,19 @@
+banners: 
+  - name: Banner 1
+    suggestion_text: "Help improve GOV.UK"
+    suggestion_link_text: "Take part in user research"
+    survey_url: https://google.com
+    page_urls: 
+      - https://google.com
```

Comment:
We should use a path here, rather than a full url. So that we have a way to review the changes before making them live. ie so the banner would show on the review app (which is on heroku), or on integration.

So you could either replicate the [previous implementation](https://github.com/alphagov/frontend/pull/3773/files) ie include the RecruitmentBanner module into the ContentItem presenter - which means you've access to the base_path from the ContentItem object. Or you could parse the `request.original_url` to extract the path. I feel like the first option is maybe cleaner? But up to you.